doc/langref.html.in

```diff
@@ -5509,6 +5525,21 @@ fn doAThing(str: []u8) void {
         // we promise that InvalidChar won't happen (or crash in debug mode if it does)
         error.InvalidChar => unreachable,
     }
+}
+      {#end_syntax_block#}
+      <p>
+      You must use the variable capture syntax. If you don't need the
+      variable, you can capture with {#syntax#}_{#endsyntax#} and avoid the
+      {#syntax#}switch{#endsyntax#}.
+      </p>
+      {#syntax_block|zig|handle_no_error_scenarios.zig#}
+fn doADifferentThing(str: []u8) void {
+    if (parseU64(str, 10)) |number| {
+        doSomethingWithNumber(number);
+    } else |_| {
+        // do as you'd like
+        unreachable;
```

Comment:
Same here, the example seems to imply that `unreachable` is necessary. Maybe change the above comment to something like `do as you'd like, for example if you know that the error condition will never happen: ...` before the `unreachable` line;

Reply:
Ah good point. I can just delete the `unreachable` line and it will probably be clear?